lxd/storage/backend_lxd.go

```diff
@@ -1387,7 +1387,7 @@ func (b *lxdBackend) RefreshCustomVolume(projectName string, srcProjectName stri
 // RefreshInstance synchronises one instance's volume (and optionally snapshots) over another.
 // Snapshots that are not present in the source but are in the destination are removed from the
 // destination if snapshots are included in the synchronisation.
-func (b *lxdBackend) RefreshInstance(inst instance.Instance, src instance.Instance, srcSnapshots []instance.Instance, allowInconsistent bool, op *operations.Operation) error {
+func (b *lxdBackend) RefreshInstance(inst instance.Instance, src instance.Instance, srcSnapshots []instance.Instance, allowInconsistent bool, snapshots bool, op *operations.Operation) error {
```

Comment:
Can we use `srcSnapshots` argument rather than needing a new argument?

Reply:
Yeah looking at where `RefreshInstance` is called, `srcSnapshots` should be nil in instance only mode. Whereas if there are no snapshots to sync it should be an empty slice I think.